src/routes/users/createUser.ts

```diff
@@ -17,12 +17,12 @@ const url = '/user';
 
 export const handler: RouteHandler<CreateUserRoute> = async (req, reply) => {
   try {
-    const { name, last_name, area_id } = req.body;
+    const { firstName, lastName } = req.body;
 
     const newUser = {
-      name,
-      last_name,
-      area_id,
+      firstName,
+      lastName,
+      // areaId,
       /** TODO: update these values with requester USER */
       CreatedBy: 'ADMIN_TEST',
       ModifiedBy: 'ADMIN_TEST',
```

Comment:
It might be a good idea to consider adding 'ADMIN_TEST' to an environment (.env) file. This approach can help protect sensitive information related to significant user roles in the database. By storing such values in an environment file, we can enhance security and better manage configurations. This is just a suggestion, and I'm open to keeping it as it is if that aligns with the project's requirements and best practices.

Last but not least, let's ensure consistent casing for key names within the codebase. I suggest changing it to "createdBy" and "modifiedBy" if this adjustment aligns with the coding standards of this library. Consistency in naming enhances code readability and maintainability.

```suggestion
      require('dotenv').config();
      createdBy: process.env.ADMIN_TEST,
      modifiedBy: process.env.ADMIN_TEST,
```

Reply:
Although the idea of moving it to a `.env` file sounds great, this is a TODO since we will use the session to take these values and insert them into each query.  @leandro0day  is working on something related to sessions/cookies to fix this.